compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -620,8 +619,14 @@ void addFunctionType(DiagnosticPos pos, Set<Whitespace> ws, boolean paramsAvail,
 
         if (paramsAvail) {
             functionTypeNode.addWS(commaWsStack.pop());
-            this.varListStack.pop().forEach(v -> functionTypeNode.params.add(v));
+            functionTypeNode.params.addAll(this.varListStack.pop());
+
```

Comment:
Extra line?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/19976/commits/7432217a84fd35c7ae7682d1e9173e9416f9ccdc